build.gradle

```diff
@@ -1,6 +1,18 @@
 plugins {
     id 'java'
     id 'application'
+    id 'idea'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.50'
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
+    id "org.springframework.boot" version "2.1.7.RELEASE"
+    id "io.spring.dependency-management" version "1.0.7.RELEASE"
```

Comment:
Are we using spring any more? I think you have removed all the Spring related code now